helix-term/src/commands/typed.rs

```diff
@@ -2246,6 +2282,13 @@ pub const TYPABLE_COMMAND_LIST: &[TypableCommand] = &[
             fun: set_option,
             completer: Some(completers::setting),
         },
+        TypableCommand {
+            name: "toggle-option",
+            aliases: &["toggle"],
+            doc: "Toggle a config option at runtime.\nFor example to toggle smart case search, use `:toggle search.smart-case`.",
+            fun: toggle_option,
+            completer: Some(completers::setting),
```

Comment:
How possible would it be to only complete _boolean_ setting names instead of all of them?
BTW, this is fantastic, I would love to see this added.

Reply:
Good idea, should be pretty trivial, all you would need to do is use `value.is_boolean()` in the `get_keys` function in `helix-term/ui/mod.rs` instead of `!value.is_object()`

Reply:
yeah, i'll have a shot

Reply:
done in a new commit